love/src/components/FlightTracker/MapFlightTracker.jsx

```diff
@@ -0,0 +1,367 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styles from './FlightTracker.module.css';
+import { isEqual } from 'lodash';
+import * as d3 from 'd3';
+import CoquimboURL from './Maps/Coquimbo.geojson';
+import ValparaisoURL from './Maps/Valparaiso.geojson';
+import AtacamaURL from './Maps/Atacama.geojson';
+import { ReactComponent as Map200 } from './Maps/Map200.svg';
+import { ReactComponent as Map160 } from './Maps/Map160.svg';
+import { ReactComponent as Map100 } from './Maps/Map100.svg';
+
+export default class MapFlightTracker extends Component {
+  static propTypes = {
+    /* Planes data with the distance to the center */
+    planes: PropTypes.arrayOf(PropTypes.objectOf({
+      id: PropTypes.string,
+      latitude: PropTypes.number,
+      longitude: PropTypes.number,
+      altitude: PropTypes.number,
+      track: PropTypes.number,
+      distance: PropTypes.number,
+      speed: PropTypes.number,
+    })),
+    /* Level of maps zoom*/
+    zoom: PropTypes.string,
+    /** Function to determine the status of the aircraft, from the distance to the radar */
+    distanceStatus: PropTypes.func,
+  };
+
+  static defaultProps = {
+    zoom: '200',
+    planes: [],
+    distanceStatus: () => {},
+  };
+
+  constructor(props) {
+    super(props);
+    this.telescopeCoords = [-70.73709442008416, -30.240476801377167];
+    this.long_lat_serena = [-71.25715298618236, -29.89192170340795];
+  }
+
+  componentDidMount = () => {
+    const { planes } = this.props;
+    planes.map((airCraft) => {
+      this.addPlanes(airCraft);
+    });
+  };
+
+  componentDidUpdate = (prevProps) => {
+    const { zoom, planes } = this.props;
+    if (!isEqual(zoom, prevProps.zoom) || !isEqual(planes, prevProps.planes)) {
```

Comment:
`zoom` param is just a string, so we should do a more simple comparison, such as:
```javascript
zoom !== prevProps.zoom
```